chainercv/evaluations/eval_semantic_segmentation_iou.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import division
+
+import numpy as np
+
+
+def calc_semantic_segmentation_confusion(pred_labels, gt_labels, n_class):
+    """Collect confusion matrix.
+
+    Args:
+        pred_label (iterable of numpy.ndarray): A collection of predicted
+            labels. The shape of a label array
+            is :math:`(H, W)`. :math:`H` and :math:`W`
+            are height and width of the label.
+            For example, This is an array of shape :math:`(N, H, W)`.
+        gt_label (iterable of numpy.ndarray): A collection of ground
+            truth label. The shape of a ground truth label array is
+            :math:`(H, W)`. The corresponding prediction label should
+            have the same shape.
+            A pixel with value "-1" will be ignored during evaluation.
+        n_class (int): The number of classes.
+
+    Returns:
+        numpy.ndarray:
+        A confusion matrix. Its shape is :math:`(n\_class, n\_class)`.
+        The :math:`(i, j)` th element corresponds to the number of times
+        a pixel with ground truth class :math:`i` is predicted
+        to be class :math:`j`.
```

Comment:
How about `the number of pixels that is labelled as class i in ground truth and predicted as class j`?